ipl/main.c

```diff
@@ -1906,17 +1906,28 @@ out:;
 	btn_wait();
 }
 
-int fix_attributes(char *path, u32 *total)
+int fix_attributes(char *path, u32 *total, u32 is_root)
 {
 	FRESULT res;
 	DIR dir;
 	u32 i = 0;
 	u32 k = 0;
 	static FILINFO fno;
 
-	// Remove archive bit for selected "root" path.
-	f_chmod(path, 0, AM_ARC);
+	if (!is_root)
```

Comment:
I will need to fix this. we shouldn't be doing attr modification twice for subdirectories.

Reply:
Change this then res = fix_attributes(path, total, **0**);